docker/Dockerfile

```diff
@@ -45,6 +45,9 @@ COPY /bot/cogs /rodhaj/bot/cogs/
 COPY /docker/start.sh /rodhaj/start.sh
 COPY /docker/wait-for /rodhaj/wait-for
 
+RUN test -f /rodhaj/bot/.env \
+    || (echo "Environment file not found; please copy .env to bot/.env"; exit 1)
```

Comment:
The idea of having the `.env` file is to only use it during the development lifecycle, and during prod, all we have to do is to pass in a pre-determined `.env` and populate the container with the env. 

Although I did say Windows development is not really supported, it should be fine to develop on as i checked and all wheels are built also for windows. Winloop is used in place of uvloop in order to aid with Windows support. Note that I haven't tested it on Windows, hence the warning

Reply:
Wait. Do you want me to remove this line or modify it somehow?

Reply:
Basically in a nutshell, I'm just saying you should remove it

Reply:
aH

Reply:
Removed the commit